src/custom/pages/Swap/SwapMod.tsx

```diff
@@ -85,7 +85,7 @@ import { formatSmart } from 'utils/format'
 import { RowSlippage } from 'components/swap/TradeSummary/RowSlippage'
 import usePrevious from 'hooks/usePrevious'
 import { StyledAppBody } from './styleds'
-import { ApplicationModal } from 'state/application/actions'
+import { ApplicationModal } from '@src/state/application/reducer'
```

Comment:
Do you have to import from `@src`? In other places you didn't
![Uploading Screenshot from 2021-12-16 14-05-03.png…]()